src/shared/validate_op.c

```diff
@@ -606,8 +600,8 @@ int OS_GetIPv4FromIPv6(const char *ip_address, char *dst_ip, size_t dst_size)
 }
 
 /* Expand IPv6 address */
-int OS_ExpandIPv6(const char *ip_address, int cidr, char *dst_ip, size_t dst_size) {
```

Comment:
Delete `const ` from `char *ip_address` allows to modify ip address, but we should be sure about if fit expanded ip into reserved space to `ip_address`.

Reply:
Fixed here https://github.com/wazuh/wazuh/pull/11268/commits/bdb7101d3eee2270e345aa0bef67324dbfc8c689